config/locales/two_factor_authentication/en.yml

```diff
@@ -137,8 +137,10 @@ en:
       backup_code_info_html: We’ll give you 10 codes. You can use backup codes as your
         only authentication method, but it is the least recommended method since
         notes can get lost. Keep them in a safe place.
+      least_secure_label: Least secure
       less_secure_label: Less secure
       more_secure_label: More Secure
+      most_secure_label: Most Secure
```

Comment:
I wonder why `i18n-tasks unused` doesn't flag this 🤔  It doesn't appear to match any of our [currently ignored patterns](https://github.com/18F/identity-idp/blob/2b73f0d8bc0f471379e519324145bdd2e29d0e49/config/i18n-tasks.yml#L92-L125). I don't think it's much of a problem; just wouldn't want to be surprised by spontaneous build failures in the future 😅